lib/libpax/libpax_api.h

```diff
@@ -31,7 +31,7 @@ struct libpax_config_t {
                                             // <-  13 ..........1 ->
                                             //    0b1010000001001 would be Channel: 1, 4, 11, 13
     uint8_t wificounter;                    // set to 0 if you do not want to install the WiFi sniffer
-    uint8_t wifi_my_country;                // e.g 0 = "EU", etc. select locale for WiFi RF settings
+    char wifi_my_country[3];                // set country code for WiFi RF settings, e.g. "01", "DE", etc.
```

Comment:
This will break `libpax_serialize_config` / `libpax_deserialize_config` for reading existing configs as all fields after `wifi_my_country` are moved by two bytes.

I see two options:

1) release as breaking change
 
 - to follow semantic versioning this should be `2.0.0` and not `1.0.2` to signal that it is a breaking change (to API as well as stored configs)
 - increase `CONFIG_MAJOR_VERSION`

2) release with backwards-compatibility

 - add a `wifi_my_country_str` (or similar) at the end of the struct and keep the `uint8_t wifi_my_country`
 - if `wifi_my_country_str` is not set read `wifi_my_country` and convert to `wifi_my_country_str`
 - reduce `libpax_config_storage_t.reserved_end` from 25 bytes to 22 to keep the size of `libpax_config_storage_t` constant
 - increase `CONFIG_MINOR_VERSION`

Reply:
I go for no 2. and added commit 0f6c9dc to keep backward compatibility.
@FlorianLudwig Please check, thanks!